VAMobile/e2e/tests/LabsAndTests.e2e.ts

```diff
@@ -53,13 +50,26 @@ beforeAll(async () => {
 })
 
 describe('Labs And Test Screen - Date Picker', () => {
-  it('month and year selection verification', async () => {
-    await element(by.id(LabsAndTestsE2eIDConstants.MONTH_PICKER_ID)).tap()
-    await element(by.text(MONTHS[3])).tap()
-    await element(by.id(LabsAndTestsE2eIDConstants.MONTH_CONFIRM_PICKER_ID)).tap()
-    await element(by.id(LabsAndTestsE2eIDConstants.YEAR_PICKER_ID)).tap()
-    await element(by.text('2020')).tap()
-    await element(by.id(LabsAndTestsE2eIDConstants.YEAR_CONFIRM_PICKER_ID)).tap()
+  beforeEach(async () => {
+    // Navigate back to Labs and Tests screen before each test
+    await navigateToLabsAndTests()
+  })
+  it('date range selection verification', async () => {
```

Comment:
BLOCKER: update test to selection a different date range to make sure its updates, `all of year`